src/screens/EventDetailsScreen/Header.js

```diff
@@ -1,36 +1,26 @@
 // @flow
 import React from "react";
-import { Text, StyleSheet, StatusBar, ImageBackground } from "react-native";
+import type { Node } from "react";
+import { StyleSheet, StatusBar, View } from "react-native";
 import SafeAreaView from "react-native-safe-area-view";
-import Touchable from "../../components/Touchable";
+import { darkBlueGreyTwoColor } from "../../constants/colors";
 
 type Props = {
-  onBackButtonPress: () => void,
-  imageUrl: string
+  children: Node
 };
 
-const Header = ({ onBackButtonPress, imageUrl }: Props) => (
-  <ImageBackground
-    style={styles.header}
-    source={{ uri: imageUrl }}
-    resizeMode="cover"
-  >
+const Header = ({ children }: Props) => (
+  <View style={styles.header}>
```

Comment:
On iOS you can tell the screen reader, that an element is a header. Maybe this should get `accessibilityTraits={["header"]}`.

See: http://facebook.github.io/react-native/docs/accessibility.html